EventListener/VersionListener.php

```diff
@@ -22,13 +21,11 @@
  */
 class VersionListener
 {
-    private $viewHandler;
     private $versionResolver;
     private $defaultVersion;
 
-    public function __construct(ViewHandlerInterface $viewHandler, VersionResolverInterface $versionResolver, $defaultVersion = null)
+    public function __construct(ViewHandlerInterface $viewHandler = null, VersionResolverInterface $versionResolver, $defaultVersion = null)
     {
-        $this->viewHandler = $viewHandler;
```

Comment:
@juillerat why? VersionExclusionListener sets version into viewHandler. Wouldn't be logical to take it here?